ssh/DOCS.md

```diff
@@ -72,6 +72,22 @@ comparison to installing any other Home Assistant add-on.
 1. Check the logs of the "Advanced SSH & Web Terminal" add-on to see if everything
    went well.
 
+<details>
+  <summary> :rotating_light: NOTE :rotating_light: </summary>
+  
+  --- 
+  
+   **RSA is deprecated**
+  
+  Please use *ed25519* instead. <br />
+  To generate a new ed25519 key, (on linux) you can use this command: <br />
+  `ssh-keygen -t ed25519` <br /><br />
+  Paste the content of your ``~/.ssh/id_ed25519.pub`` file in the `authorized_keys` section
```

Comment:
- I don't think we have to add rotating lights here.
- We have links to external instructions down below. IMHO, it is odd to list those here (especially since this isn't needed for running or installing the add-on).

I suggest removing this details block at full./

../Frenck

Reply:
These instructions are not needed and not complete, and in that case, IMHO causes even more hair pulling. The correct instructions are linked below, therefore, these make no sense to have here.

Please remove this part.